### PR TITLE
Update mod.conf

### PR DESCRIPTION
**Pull request by MilesBDyson**
updated mod.conf file please review

fixes deprecation warnings

### CODE REVIEW

File: mod.conf [hunk 1]
```diff
@@ -1 +1,8 @@
 name = stamina
+author =
```
Comment (SmallJoker) [inline mod.conf]: sofar and various contributors

File: init.lua [hunk 2]
```diff
@@ -329,7 +329,7 @@ local function move_tick()
 	for _,player in ipairs(minetest.get_connected_players()) do
 		local controls = player:get_player_control()
 		local is_moving = controls.up or controls.down or controls.left or controls.right
-		local velocity = player:get_player_velocity()
+		local velocity = player:get_velocity()
```
Comment (SmallJoker) [inline init.lua:332]: MInetest requirement `5.4.0`

File: mod.conf [hunk 3]
```diff
@@ -1 +1,8 @@
 name = stamina
+author = 
+description = Adds stamina and hunger effects.
+min_minetest_version = 5.0
+depends = default
+optional_depends = 3d_armor,player_monoids,
+version = ?
```
Comment (SmallJoker) [inline mod.conf]: It's kind of rolling release right now. Inaccurate and superfluous field.

**Comment by SmallJoker** [comment]
While you're at it. Please remove `depends.txt` and `description.txt` to supersede #27.

**Comment by MilesBDyson** [comment]
that should do it then.

**Comment by SmallJoker** [comment]
What about `player:get_velocity()`? Prior to 5.4.0, that function returned `nil` on players. If you want to get rid of deprecation messages, check against the presence of `minetest.features.direct_velocity_on_players` to decide whether to use `get_player_velocity` or `get_velocity`.

File: mod.conf [hunk 4]
```diff
@@ -1 +1,8 @@
 name = stamina
+author = Various
+description = Adds stamina and hunger effects.
+min_minetest_version = 5.0
```
Comment (S-S-X) [inline mod.conf:4]: Just add last 5.0 supported release (git tag), bump this to 5.4 and let legacy users to use previous stable and usable version?
Allows using latest API docs without thinking about compatibility and allows fixing deprecated stuff without thinking about breaking something.

Also if needed possible bugs on old versions can still be fixed easily if someone wants to do that, it is git after all.

**Comment by SmallJoker** [comment]
Outdated and abandoned.

**Pull request closed without merging**